params/config.go

```diff
@@ -431,8 +431,22 @@ var NetworkNames = map[string]string{
 
 // Arbitrum
 // ArbOSInit defines some initialization values for ArbOS state.
+// lint:require-exhaustive-initialization
 type ArbOSInit struct {
-	NativeTokenSupplyManagementEnabled bool `json:"nativeTokenSupplyManagementEnabled"`
+	NativeTokenSupplyManagementEnabled bool     `json:"nativeTokenSupplyManagementEnabled"`
+	InitialL1BaseFee                   *big.Int `json:"initialL1BaseFee,omitempty"`
+}
+
+// The initial L1 pricing basefee starts at 50 GWei unless set in the init message
+var DefaultInitialL1BaseFee = big.NewInt(50 * GWei)
+
+// GetInitialL1BaseFee returns the initial L1 base fee for the ArbOS instance.
+// If not set, it returns the default value (backwards compatibility).
```

Comment:
nitpick: Regarding "backwards compatibility". 
It is ambiguous to me.
When looking at the changes introduced by the PR I can infer that it means that InitialL1BaseFee was not present when ArbOSInit was added.
But only reading the code this is not clear, and can raise questions if it means keeping this geth version compatible with older nitro versions.
That said I would remove this "backwards compatibility" part.